src/mocks/browser-handlers.ts

```diff
@@ -18,6 +19,7 @@ const handlers = [
 	...dataTypeHandlers,
 	...documentTypeHandlers,
 	...treeHandlers,
+	...propertyEditorHandlers,
```

Comment:
This should probably be added to `e2e-handlers.ts` as well so we have some mocked property editors in Playwright.

Reply:
This is updated now 👍